modules/components/pages/outputs/redpanda_migrator_offsets.adoc

```diff
@@ -501,11 +430,82 @@ A role ARN to assume.
 
 An external ID to provide when assuming a role.
 
-
 *Type*: `string`
 
 *Default*: `""`
 
+
+== metadata_max_age
+
+The maximum period of time after which metadata is refreshed.
+
+*Type*: `string`
+
+*Default*: `5m`
+
+
+=== `kafka_key`
+
+The Kafka key of a message.
```

Comment:
Is this description correct?

Reply:
I checked and this is the only place that this field name and description appears. I've updated the text as suggested.